kalamine/msklc_manager.py

```diff
@@ -25,48 +23,36 @@ def __init__(
         self._verbose = verbose
         self._working_dir = working_dir
 
-    # This need the klc file to be created first
     def create_c_files(self):
-        """Call kbdutools to generate C files"""
-        kbdutools = self._msklc_dir / Path("bin/i386/kbdutool.exe")
+        """Call kbdutool on the KLC descriptor to generate C files."""
+
+        kbdutool = self._msklc_dir / Path("bin/i386/kbdutool.exe")
         cur = os.getcwd()
         os.chdir(self._working_dir)
         ret = subprocess.run(
-            [kbdutools, "-u", "-s", self._layout.meta["name8"] + ".klc"],
+            [kbdutool, "-u", "-s", f"{self._layout.meta['name8']}.klc"],
             capture_output=not self._verbose,
         )
         os.chdir(cur)
         ret.check_returncode()
 
-    # Check if the driver is already installed
-    # as this will cause MSKLC to launch the GUI instead of creating the installer
     def _is_already_installed(self) -> bool:
+        """Check if the keyboard driver is already installed,
+        which would cause MSKLC to launch the GUI instead of creating the installer."""
+
         sys32 = Path(os.environ["WINDIR"]) / Path("System32")
         dll = sys32 / Path(f'{self._layout.meta["name8"]}.dll')
         return dll.exists()
 
     def _create_dummy_layout(self) -> str:
-        dummy_toml = create_layout_content(
-            self._layout.geometry, self._layout.has_altgr, self._layout.has_1dk
-        )
-        dummy_toml = dummy_toml.replace(
-            "custom QWERTY layout", self._layout.meta["description"]
-        )
-        dummy_toml = dummy_toml.replace("qwerty-custom", self._layout.meta["name"])
-        dummy_toml = dummy_toml.replace("custom", self._layout.meta["name8"])
-        dummy_toml = dummy_toml.replace("nobody", self._layout.meta["author"])
-        dummy_toml = dummy_toml.replace(
-            "https://OneDeadKey.github.com/kalamine", self._layout.meta["url"]
-        )
-        dummy_toml = dummy_toml.replace("0.0.1", self._layout.meta["version"])
-        dummy_toml = dummy_toml.replace('"us"', f'"{self._layout.meta["locale"]}"')
```

Comment:
This line should’ve triggered a red flag by me : replacing a two-char string all over the place brings bad luck. :-)

Even the `custom` string has raised real-life issues on my own keyboard layout descriptors.